Dockerfile

```diff
@@ -1,54 +1,18 @@
-FROM ubuntu:eoan AS dawn-gcc9-base-env
-RUN apt update && apt install -y \
-    build-essential ninja-build cmake \
-    llvm-9-dev libclang-9-dev \
-    python3 python3-pip libpython-dev \
-    python3-setuptools python3-wheel \
-    libboost-dev git curl && apt clean
-RUN python3 -m pip install --upgrade pip
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
-
-FROM dawn-gcc9-base-env AS dawn-gcc9-env
-RUN curl -L https://github.com/protocolbuffers/protobuf/releases/download/v3.10.1/protobuf-all-3.10.1.tar.gz | \
-    tar -xz -C /usr/src
-RUN mkdir -p /usr/src/protobuf-3.10.1/build
-RUN cmake -S /usr/src/protobuf-3.10.1/cmake -B /usr/src/protobuf-3.10.1/build \
-    -Dprotobuf_BUILD_EXAMPLES=OFF \
-    -Dprotobuf_BUILD_TESTS=OFF \
-    -Dprotobuf_INSTALL_EXAMPLES=OFF \
-    -Dprotobuf_BUILD_PROTOC_BINARIES=ON \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DCMAKE_INSTALL_PREFIX=/usr/local \
-    -DBUILD_SHARED_LIBS=ON \
-    -GNinja
-RUN cmake --build /usr/src/protobuf-3.10.1/build --target install -j $(nproc)
-RUN rm -rf /usr/src/protobuf-3.10.1/build
-RUN cd /usr/src/protobuf-3.10.1/python && python setup.py build
-RUN mv /usr/src/protobuf-3.10.1/python/build/lib/google /usr/local/lib/google
-RUN curl -L https://github.com/GridTools/gridtools/archive/v1.0.4.tar.gz | \
-    tar -xz -C /usr/src
-RUN mkdir -p /usr/src/gridtools-1.0.4/build
-RUN cmake -S /usr/src/gridtools-1.0.4 -B /usr/src/gridtools-1.0.4/build \
-    -DBUILD_TESTING=OFF \
-    -DINSTALL_TOOLS=OFF \
-    -DGT_INSTALL_EXAMPLES=OFF \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DCMAKE_INSTALL_PREFIX=/usr/local \
-    -GNinja
-RUN cmake --build /usr/src/gridtools-1.0.4/build -j $(nproc) --target install
-RUN rm -rf /usr/src/gridtools-1.0.4/build
-
-FROM dawn-gcc9-env AS dawn-gcc9
+# IMAGE needs be be set to one of the docker/dawn-env.dockerfile images
+ARG IMAGE=gtclang/dawn-env-cuda10.1-ubuntu18.04
```

Comment:
if we now have a docker folder, should we put this file also in there (and adapt the path on dockerhub?)

Reply:
I'm not sure... Sometimes it's nice to have a top-level `Dockerfile` that people can immediately build. Something like a `Makefile` used to be.

Reply:
But I'm open to either way! Let me know.